packages/vue-app/template/client.js

```diff
@@ -37,10 +37,10 @@ const NUXT = window.<%= globals.context %> || {}
 
 Object.assign(Vue.config, <%= serialize(vue.config) %>)<%= isTest ? '// eslint-disable-line' : '' %>
 
-<% if (isDev) { %>
+<% if (options.render.ssrLog) { %>
```

Comment:
Displaying SSR logs for production is dangerous anyway. Can lead to potential data leak.

Reply:
PS: With request concurrency in prod logs can be shared between users :)

Reply:
It's disabled by default

Reply:
I see. But for the technical reason above, it is non-sense at all to have this functionality for production

Reply:
```suggestion
<% if (options.render.ssrLog && isDev) { %>
```

Agree with @pi0 here.